test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs

```diff
@@ -36,7 +36,7 @@ public Task ResponseCompression_Windows_NoCompression(ServerType serverType, Run
 
         [Theory]
         [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, "http://localhost:5102/", ApplicationType.Portable)]
-        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, "http://localhost:5103/", ApplicationType.Standalone)]
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, "http://localhost:5103/", ApplicationType.Standalone, Skip = "https://github.com/aspnet/ServerTests/issues/53")]
```

Comment:
You'll probably have to comment out because of https://github.com/xunit/xunit/issues/1087